### ISSUE TITLE
Bind AmmoCount and DefaultClip functions

### ISSUE THREAD

**Issue by Kefta** [Facepunch/garrysmod-requests#703]
As defined in basecombatweapon_shared.h:
/////////////////////////////////////
// Ammo quantity queries for weapons that do not use clips. These are only
// used to determine how much ammo is in a weapon that does not have an owner.
// That is, a weapon that's on the ground for the player to get ammo out of.
int GetPrimaryAmmoCount() { return m_iPrimaryAmmoCount; }
void SetPrimaryAmmoCount( int count ) { m_iPrimaryAmmoCount = count; }

int GetSecondaryAmmoCount() { return m_iSecondaryAmmoCount; }
void SetSecondaryAmmoCount( int count ) { m_iSecondaryAmmoCount = count; }
/////////////////////////////////////
tl;dr, Used to get reserve ammo of weapon when on the ground. CS:S uses this to give bonus ammo based on a map firing an amount to the weapon entity.

(Get/Set)DefaultClip(1/2) gets the weapon default clip value
